util/lua/lua_test.go

```diff
@@ -868,7 +868,7 @@ return hs`
 	})
 
 	t.Run("Get resource health for */* override with empty health.lua", func(t *testing.T) {
-		testObj := StrToUnstructured(ec2AWSCrossplaneObjJSON)
+		testObj := StrToUnstructured(objWithNoScriptJSON)
```

Comment:
Had to switch to a non-crossplane resource, since that's now supported by the built-in wildcard health check.